src/targets/gpu/mlir.cpp

```diff
@@ -641,6 +697,34 @@ void adjust_param_shapes(module& m, const std::vector<instruction_ref>& inputs)
     }
 }
 
+static void update_tuning_table(MlirRockTuningTable& tuning_table)
```

Comment:
This function should return the `tuning_table` and be named `create_tuning_table`.

```cpp
static mlir_tuning_table create_tuning_table()
{
...
}
```